test-data/unit/check-inference.test

```diff
@@ -1976,7 +1976,7 @@ class C:
 
 if bool():
     f()
-    1 + '' # E: Unsupported left operand type for + ("int")
+    1 + '' # E: Unsupported operand types for + ("int" and "str")
```

Comment:
Similar to above, I'd prefer using `1()` instead (and not changing the fixture).